### ISSUE TITLE
customPrismaImportPath isn't used for imports when output location of prisma-client-js generator includes `node_modules`

### ISSUE THREAD

**Issue by mattkindy** [MichalLytek/typegraphql-prisma#270]
**Describe the Bug**
If you decide to use `customPrismaImportPath` configuration option, but your `prisma-client-js` is being output to a custom location which is within `node_modules`, the Prisma imports default to `@prisma/client` instead of using the `customPrismaImportPath`.

**To Reproduce**
Given the following `schema.prisma`, 

```prisma
generator client {
  provider = "prisma-client-js"
  output   = "./node_modules/@generated/prisma-client"
}

generator typegraphql {
  provider = "typegraphql-prisma"
  customPrismaImportPath = "../prisma-client"
}
```

Prisma imports will refer to `@prisma/client`

**Expected Behavior**
All TypeGraphQL Prisma code should import Prisma using `customPrismaImportPath` if specified.

**Environment (please complete the following information):**

- OS: macOS Big Sur 11.6
- Node 14.18.3
- `typegraphql-prisma` version 0.20.0
- Prisma version 3.12.0
- TypeScript version 4.6.3

**Comment by mattkindy** [comment]
Relevant code snippets:

**`src/cli/prisma-generator.ts`**
```ts
  const internalConfig: InternalGeneratorOptions = {
    outputDirPath: outputDir,
    relativePrismaOutputPath: toUnixPath(
      path.relative(outputDir, prismaClientPath),
    ),
    absolutePrismaOutputPath: prismaClientPath.includes("node_modules")
      ? "@prisma/client"
      : undefined,
  };
```

**`src/generator/imports.ts`**
```ts
export function generatePrismaNamespaceImport(
  sourceFile: SourceFile,
  options: GeneratorOptions,
  level = 0,
) {
  sourceFile.addImportDeclaration({
    moduleSpecifier:
      options.absolutePrismaOutputPath ??
      (level === 0 ? "./" : "") +
        path.posix.join(
          ...Array(level).fill(".."),
          options.customPrismaImportPath ?? options.relativePrismaOutputPath,
        ),
    namedImports: ["Prisma"],
  });
}
```

**Comment by zoheb391** [comment]
bump any update here? also facing this issue

**Comment by zoheb391** [comment]
hi @MichalLytek, any update on this?

**Comment by MichalLytek** [comment]
Fixed by bfe88cd 🔒
